sdk/openai/openai/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 ### Features Added
 
+- `getAudioTranscription` and `getAudioTranslation` now take a `StreamProducer` as input. This allows for streaming audio files to the service instead of having to load the entire file into memory. For example, an audio file saved on desk can be streamed to the service in NodeJS as follows:
```

Comment:
```suggestion
- `getAudioTranscription` and `getAudioTranslation` now take a `StreamProducer` as input. This allows for streaming audio files to the service instead of having to load the entire file into memory. For example, an audio file saved on disk can be streamed to the service in NodeJS as follows:
```